core/src/main/scala/jms4s/jms/JmsContext.scala

```diff
@@ -88,12 +109,22 @@ class JmsContext[F[_]: Async: Logger](private val context: JMSContext) {
   private def createQueue(queue: QueueName): F[JmsQueue] =
     Sync[F].blocking(context.createQueue(queue.value)).map(new JmsQueue(_))
 
+  private def createTemporaryQueue(): F[JmsQueue] =
+    Sync[F]
+      .blocking(context.createTemporaryQueue())
+      .map(new JmsQueue(_))
+
   private def createTopic(topicName: TopicName): F[JmsTopic] =
     Sync[F].blocking(context.createTopic(topicName.value)).map(new JmsTopic(_))
 
-  def createDestination(destination: DestinationName): F[JmsDestination] = destination match {
-    case q: QueueName => createQueue(q).widen[JmsDestination]
-    case t: TopicName => createTopic(t).widen[JmsDestination]
+  private def createTemporaryTopic(): F[JmsTopic] =
+    Sync[F].blocking(context.createTemporaryTopic()).map(new JmsTopic(_))
+
+  override def createDestination(destination: Destination): F[JmsDestination] = destination match {
+    case TemporaryQueue(_) => createTemporaryQueue().widen[JmsDestination]
```

Comment:
hi @adamretter i'm giving a closer look at the PR 🙂 
it seems that the UUID value inside `TemporaryQueue` and `TemporaryTopic` is never used outside of the .`toString`, and i can't really see were this UUID(and the toString) can be usefull in a config class that is used to create the actual JMS TemporaryQueue.
like here, it's ignored.
I'm missing something or we can remove it and simplify the api?

Reply:
@faustin0 It was previously used by a Logger to help understand the message queues that are created. It is no longer used and can now be removed. Apart from that do you have any other comments for me, or are you happy with the changes?